app/config/config.yml

```diff
@@ -7,11 +7,38 @@ framework:
         resource: "%kernel.root_dir%/config/routing.yml"
         strict_requirements: ~
     form:            ~
+    validation:
+        enable_annotations: true
     csrf_protection: ~
     templating:
         engines: ['twig']
     session:
       handler_id: ~
+      storage_id: session.storage.php_bridge
+
+# Doctrine Configuration
+doctrine:
+  dbal:
+    driver: pdo_%database.driver%
+    host: %database.host%
+    port: %database.port%
+    dbname: %database.name%
+    user: %database.user%
+    password: %database.password%
+    charset: utf8mb4
+    default_table_options:
+      charset: utf8mb4
+      collate: utf8mb4_unicode_ci
+    mapping_types:
+      # Remove when we don't have legacy tables anymore
+      enum: string
+    types:
+      enum_bool: Backend\Doctrine\Type\EnumBoolType
+      language: Backend\Doctrine\Type\LanguageType
+
+  orm:
```

Comment:
doesn't it create a default connection by default if you only enter one connection ?